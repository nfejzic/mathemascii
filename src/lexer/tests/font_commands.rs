use crate::lexer::tests::Snapshot;

use super::super::*;

super::test_snap!(font_bold, "bb bbb mathbf mathbb");
super::test_snap!(various_commands, "cc mathtt mathcal fr sf mathfrak");
