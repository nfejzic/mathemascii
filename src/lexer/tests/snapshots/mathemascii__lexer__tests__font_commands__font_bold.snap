---
source: src/lexer/tests/font_commands.rs
expression: "Snapshot((input, tokens))"
---
bb bbb mathbf mathbb

bb
^^ -> FontCommand(Bold) at: 0 -> 2

   bbb
   ^^^ -> FontCommand(BlackboardBold) at: 3 -> 6

       mathbf
       ^^^^^^ -> FontCommand(Bold) at: 7 -> 13

              mathbb
              ^^^^^^ -> FontCommand(BlackboardBold) at: 14 -> 20


